Formula/c/couchbase-shell.rb

```diff
@@ -5,8 +5,8 @@ class CouchbaseShell < Formula
   head "https://github.com/couchbaselabs/couchbase-shell.git", branch: "main"
 
   stable do
-    url "https://github.com/couchbaselabs/couchbase-shell/archive/refs/tags/v0.75.2.tar.gz"
-    sha256 "72d99bf1de8a050137b080825299eca01d2aa4fb5e10bf75927008314e88b783"
+    url "https://github.com/couchbaselabs/couchbase-shell/archive/refs/tags/v1.0.0.tar.gz"
+    sha256 "404c704f5816c3abd26d460ecbd2e049e45170854948a7219cd9ec41a6fb753c"
 
     # Use `llvm@15` to work around build failure with LLVM Clang 16 (Apple Clang 15)
```

Comment:
Does this need manually removing by us, or will you do it @chenrui333

Reply:
it would be manually done

Reply:
After this is merged? Or can I raise aPR that bumps to 1.0.0 and removes the patch?